ceph/ceph.py

```diff
@@ -1559,9 +1559,9 @@ def long_running(self, **kw):
         long_running = kw.get("long_running", False)
         if "timeout" in kw:
             timeout = None if kw["timeout"] == "notimeout" else kw["timeout"]
-        else:
-            # Set defaults if long_running then 1h else 5m
-            timeout = 3600 if kw.get("long_running", False) in kw else 300
+        if long_running:
+            # Set timeout to 1h if long_running is enabled
+            timeout = 3600
```

Comment:
This would break blocking commands default timeout from being set to `5mins`

Reply:
The latest change would not honor `timeout` value if set when `long_running=True` as it would revert to defaults `i.e. 3600`.